ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -40071,42 +40070,33 @@ periodics:
         secretName: ci-pull-credentials
     - name: cluster-profile
       secret:
-        secretName: cluster-secrets-equinix-ocp-metal
+        secretName: cluster-secrets-packet-assisted
     - name: pull-secret
       secret:
         secretName: registry-pull-credentials
     - name: result-aggregator
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build04
+  cluster: build05
+  cron: 0 0 * * 1
   decorate: true
   decoration_config:
     skip_cloning: true
   extra_refs:
   - base_ref: master
     org: openshift
     repo: release
-  interval: 24h
   labels:
-    ci-operator.openshift.io/cloud: packet-edge
-    ci-operator.openshift.io/cloud-cluster-profile: packet-sno
+    ci-operator.openshift.io/cloud: equinix-ocp-metal
+    ci-operator.openshift.io/cloud-cluster-profile: equinix-ocp-metal
+    ci-operator.openshift.io/cluster: build05
     ci-operator.openshift.io/variant: nightly-4.12
     ci.openshift.io/generator: prowgen
     ci.openshift.io/no-builds: "true"
     job-release: "4.12"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-release-master-nightly-4.12-e2e-metal-ovn-single-node-live-iso
-  reporter_config:
-    slack:
-      channel: '#single-node-ci'
-      job_states_to_report:
-      - success
-      - failure
-      - error
-      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
-        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
-        <{{.Status.URL}}|View logs>'
```

Comment:
this one needs to come back (``make update/jobs/etc`` sometimes "forgets" about keeping those configurations)